### PR TITLE
Updates to index and en-Contribution

### LINKED ISSUE #314: Re-structure and update Contributing
As discussed here https://github.com/jamulussoftware/jamuluswebsite/discussions/294

Main website [en-Contributing.md](https://jamulus.io/wiki/Contribution) - make this a brief “overview” page for both code and non-code contribution, linking to further information here:

**Jamulus main repo**

CONTRIBUTING.md - make this the guide for engineers wishing to contribute code. Link to the above overview at the end. Note that the file name is by convention (despite it also having a meaning of monetary contribution), but we should try to say “get involved” or similar when linking to it.

README.md - have this contain a simple link to the overview page only.

**Jamulus website repo**

README.md - make this the guide for people wanting to use the repo to contribute documentation or translation (eg use of Jekyll, include files etc..  Link to the code contribution page from here.

CONTRIBUTING.md - where any "project management" processes should be documented. Links to README.md above [in practice this may come out weirdly separated from the readme, but I'll try my best

In terms of the level of detail, I'd say keep it fairly general. If people have questions they can ask and we can add to these files if necessary in response to that. I’d rather people had too little so that they would be encouraged to engage than so much that they would be encouraged to go it alone and get it wrong.

### PR DESCRIPTION
**Pull request by gilgongo**
# Does this need translation?

- [x] Yes 
- [ ] No 

# Changes

Shorten index text as per discussion https://github.com/jamulussoftware/jamuluswebsite/issues/322#issuecomment-799796098

Make website contributing a general overview.

### CODE REVIEW

File: 1-index.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
   <div class="fx-col-100-xs">
     <div itemprop="abstract">
       <h2>What is Jamulus?</h2>
-      Jamulus is software for playing music, rehearsing, or just jamming with anyone online with low latency. You can use your Windows, macOS or Linux machine to connect to Jamulus servers worldwide. Jamulus is free and you can just use your normal broadband connection. Simply connect to a public server or host your own private one. Jamulus has been in development since 2006 and is designed for high quality, low-latency sound, making it easy to play together remotely and in time. Have a look at <a href="wiki/Demos" target="_blank" rel="noreferrer">how people do it</a>!
+       Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. With high quality, low-latency sound on a normal broadband connection, it's easy to play together remotely and in time. Have a look at <a href="wiki/Demos" target="_blank" rel="noreferrer">how people do it</a> - and  <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">download it here</a>!
```
Comment (ann0see) [inline 1-index.html:20]: Sounds good. However we‘re not linking to the download page here? So wouldn’t it be better to say get started here?

File: wiki/en/en-Contribution.md [hunk 2]
```diff
@@ -7,32 +7,13 @@ permalink: "/wiki/Contribution"
...
+If you think some documentation or information is missing or can be improved, <a href="https://github.com/jamulussoftware/jamulus/discussions">post about that on this forum</a> so that it can be discussed first.
 
-See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for further information.
+If you want to help out with translation, you will need to have a GitHub account and be familiar with using git and Jekyll. Read about our [documentation and translation process](https://github.com/jamulussoftware/jamuluswebsite/blob/changes/CONTRIBUTING.md), and [introduce yourself](https://github.com/jamulussoftware/jamulus/discussions) if you’d like to get involved.
```
Comment (ann0see) [inline wiki/en/en-Contribution.md]: Maybe: „should be familiar with git“ we can probably also help people with git?

File: 1-index.html (hunk 1 above)
Comment (gilgongo) [reply 1-index.html:20]: I thought we'd link to that until we had the download page. It's got the download buttons on it.

File: wiki/en/en-Contribution.md (hunk 2 above)
Comment (gilgongo) [reply wiki/en/en-Contribution.md]: OK sure.

File: 1-index.html (hunk 1 above)
Comment (ann0see) [reply 1-index.html:20]: Ok. So happy to approve.

**Review by ann0see — APPROVED** [review]

**Pull request merged**